src/Ushahidi/Modules/V5/Actions/Tos/Commands/CreateTosCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Actions\Tos\Commands;
+
+use App\Bus\Command\Command;
+
+class CreateTosCommand implements Command
+{
+    /**
+     * @var array
+     */
+    private $input;
```

Comment:
Lets add the fields to command rather than keeping them in array. It would make it easier to work with.